client/src/app/utils/model-utils.tsx

```diff
@@ -115,30 +115,33 @@ export const toIJobFunctionDropdownOptionWithValue = (
   toString: () => value.name,
 });
 
-// TagType
+// TagCategory
 
-export interface ITagTypeDropdown extends Pick<TagType, "id" | "name"> {}
+export interface ITagCategoryDropdown
+  extends Pick<TagCategory, "id" | "name"> {}
 
-export const toITagTypeDropdown = (value: TagType): ITagTypeDropdown => ({
+export const toITagCategoryDropdown = (
+  value: TagCategory
+): ITagCategoryDropdown => ({
   id: value.id,
   name: value.name,
 });
 
-export const toITagTypeDropdownOptionWithValue = (
-  value: ITagTypeDropdown
-): OptionWithValue<ITagTypeDropdown> => ({
+export const toITagCategoryDropdownOptionWithValue = (
+  value: ITagCategoryDropdown
+): OptionWithValue<ITagCategoryDropdown> => ({
   value,
   toString: () => value.name,
 });
 
 // Tag
 
-export interface ITagDropdown extends Pick<Tag, "id" | "name" | "tagType"> {}
+export interface ITagDropdown extends Pick<Tag, "id" | "name" | "category"> {}
```

Comment:
Why not tagCategory?

Reply:
Because the backend has changed the Tag model.